### PR TITLE
[JA] translations privacy-sandbox

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
- Update Japanese versions of the Glossary, Overview and Trust Tokens articles.
- Add Japanese version of  Summary Reports article.

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
@agektmr Ready with the translations, but it appears the build is failing on a file that is not affected by this PR.

**Comment by agektmr** [comment]
Hi @ilyaspiridonov,
Looks like there's an error in the fragment of the branch. Could you please rebase it with the main branch?

**Comment by ilyaspiridonov** [comment]
> Hi @ilyaspiridonov,
> Looks like there's an error in the fragment of the branch. Could you please rebase it with the main branch?

Done, thank you!

File: site/ja/docs/privacy-sandbox/overview/index.md [hunk 1]
```diff
@@ -1,106 +1,152 @@
...
+### プライバシーサンドボックス提案の Explainer {: #explainers }
+
+API 提案の Explainer では、特に欠落しているユースケースや、目標を達成するためのよりプライベートな方法の提案などのフィードバックを求めています。 各 Exapliner の［Issues］タブでコメントや質問を投稿できます。
+
+* [プライバシーバジェット](https://github.com/bslassey/privacy-budget)
+* [トラストトークン](https://github.com/dvorak42/trust-token-api)
+* [ファーストパーティセット](https://github.com/privacycg/first-party-sets)
+* [Gnatcatcher](https://github.com/bslassey/ip-blindness)
+* [Aggregated Reporting API](https://github.com/csharrison/aggregate-reporting-api)
+* [Attribution Reporting](https://github.com/csharrison/conversion-measurement-api)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/overview/index.md:128]:
```suggestion
* [アトリビューション レポート](https://github.com/csharrison/conversion-measurement-api)
```

**Review by agektmr — APPROVED** [review]
Requested several changes, otherwise LGTM.

**Comment by agektmr** [comment]
@ilyaspiridonov can you please merge requested changes?

**Comment by ilyaspiridonov** [comment]
@agektmr Done, thank you!

**Pull request merged**